Bottom: 65e3c506047f4fbfd7e90791f614155d37725f2d
Top:    e7cb574b8f6857c32e1cf2db70b19065f5d9efc1
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 16:54:29 +0000

Refresh of find-c-kill-setthing-with-fire

---

diff --git a/src/find.c b/src/find.c
index 0dd88bb..67e792c 100644
--- a/src/find.c
+++ b/src/find.c
@@ -401,7 +401,6 @@ PinLineIntersect (PinType *PV, LineType *Line)
 bool
 SetThing (int type, void *ptr1, void *ptr2, void *ptr3)
 {
-  assert (! (type == PIN_TYPE && ptr1 == NULL));
   thing_ptr1 = ptr1;
   thing_ptr2 = ptr2;
   thing_ptr3 = ptr3;
